test/test_suite.cpp

```diff
@@ -44,11 +44,11 @@
 #include <iostream>
 #include <iomanip>
 
-#if defined(_WIN32)
+#if defined(__unix__) || defined(__unix) || (defined(__APPLE__) && defined(__MACH__)) || defined(__MINGW32__)
```

Comment:
fixes error: 
`error: conflicting declaration 'typedef SSIZE_T ssize_t'` on msys2 / mingw32
apparrently macro _WIN32 is defined on msys2 / mingw32´ but you may not 
`typedef SSIZE_T ssize_t;`